### DISCUSSION TITLE
How to remove Theme Samples ?

### DISCUSSION THREAD

**Discussion by TechAspirant** [serenity-is/Serenity#5455] [Help] [answered]
How can I remove Theme Samples in Serenity Core ? I cannot see the AdminLte folder.

**Comment by VictorTomaili** [comment, marked as answer]
Remove this line https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Core/Initialization/Startup.cs#L53 for hide from menu. 
if you want to remove sample completely, uninstall `Serenity.Demo.ThemeSamples` nuget package.

**Reply by TechAspirant** [reply]
Thanks.
